Create nami.md (#88)
amplitude/amplitude-dev-center: docs/data/sources/nami.md

---
title: Import Nami Survey Data
description: Nami provides a complete solution for in-app purchases and subscriptions including designing no-code paywalls, campaigns to convert and retain your customers, subscription management tools, and real-time revenue analytics. 
---

!!!tip

    This integration is maintained by Nami. Contact the [Nami support team](https://www.namiml.com/) with any questions about this integration. 

## About Nami

[Nami](https://Nami.com/) is a platform to help you launch, scale, and optimize your mobile app business. Nami provides a complete solution for in-app purchases and subscriptions including designing no-code paywalls, campaigns to convert and retain your customers, subscription management tools, and real-time revenue analytics. 

## Considerations

- You must enable this integration in each Amplitude project you want to use it in.
- You need a paid plan with Nami to enable this integration.

> - Before setting up this integration, please make sure that you are identifying your users in Nami using the same User_Ids that you also use in Amplitude. using the same User_Ids in both solutions. If your user IDs don’t match up, Amplitude won’t be able to match and merge survey response data with product usage data of the same user.

Reviewer: ```suggestion
- Make sure that you indentify users in Nami using the same user identifier you use in Amplitude. If the user IDs don’t match, Amplitude can't match and merge survey response data with product use data of the same user.
```

## Amplitude setup

1. Copy the Amplitude **API key** for your project. There are no other setup steps in Amplitude. 

## Nami setup

1. Log into your Nami.io dashboard.
2. Navigate to Integrations and click **Connect to Amplitude**.
3. Paste your Amplitude API Key.
4. Choose which events you want to receive.

> ## Use Cases

Reviewer: ```suggestion
## Use cases
```

Specific use cases include:

> 1. **In-App Monetization Optimization:** Nami empowers mobile app businesses to fine-tune their in-app monetization strategies. With Nami's user-friendly no-code paywall design tools, businesses can create and customize in-app purchase and subscription offerings. This flexibility enables experimentation with pricing models, content access levels, and promotional offers. The real-time revenue analytics provided by Nami offer a comprehensive view of financial performance, helping businesses identify which monetization strategies are most effective. This data-driven approach allows for continuous refinement of monetization tactics, ultimately maximizing revenue generation.

Reviewer: @Brandoncharleskhoo1 We need to trim these down, it sounds like it's just marketing copy from Nami. Did this come from them or did you write it?

Author: Sure thing. We can trip them down. I can get them to review the marketing copy now.
